shared/js/background/utils.es6.js

```diff
@@ -2,7 +2,7 @@ const tldjs = require('tldjs')
 const entityMap = require('../../data/tracker_lists/entityMap')
 
 function extractHostFromURL (url, shouldKeepWWW) {
-    if (!url) return
+    if (!url) return ''
```

Comment:
This was erroring out in the empty URL case. Not sure if it was ever hit in production, but I think it's worth maintaining consistency between different outputs.

(E.g. if a caller expects a function to return a string, always try and return a string.)